CONTRIBUTING.md

```diff
@@ -50,11 +50,19 @@ include BREAKING CHANGE keyword for breaking changes.
 
 The message summary should be a one-sentence description of the change, and it must be 72 characters in length or shorter. For a list of tags, please [click here](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum). We use the angular style for tags. See the [default release rules](https://github.com/semantic-release/commit-analyzer/blob/master/lib/default-release-rules.js) based on the commit tag. Shown below is an example of the release type that will be done based on a commit message.
 
+In most cases, commits will be a `feat` or `fix`. Make sure to include the `BREAKING CHANGE` string in the summary if there are non-backwards-compatible changes in the commit.
+
+The tags for `refactor`, `chore`, `perf`, `docs` signal that there are **no external changes to _any_ APIs** (including non-breaking ones), and **release notes are not generated** for these tags.
```

Comment:
`test` fits into which group. guessing this one?

we don't use `improvement` but that might be one of the available options. I'm guessing it'd go into the former category and appear in the changelog (since we don't use it, I don't think we need to add it except to be complete w.r.t. our config)

Reply:
I originally didn't want to pull in every value from the standard in case it gets out of sync, but we may as well.

Reply:
We can also consider adding this: https://commitlint.js.org/#/guides-use-prompt

Reply:
I've been using commitizen locally. not sure how configurable this CLI is, but the benefit of something like commitizen is that we could configure the scope to our taste if we wished. Worth discussing.